cmd/traffic/cmd/manager/internal/mutator/service.go

```diff
@@ -90,7 +93,15 @@ func ServeMutator(ctx context.Context) error {
 	})
 	mux.HandleFunc("/uninstall", func(w http.ResponseWriter, r *http.Request) {
 		ctx := r.Context()
-		dlog.Debug(ctx, "Received uninstall request...")
+		dlog.Info(ctx, "Received uninstall request...")
+		q := r.URL.Query()
+		if rev, err := strconv.Atoi(q.Get("revision")); err != nil || rev != env.HelmRevision {
+			dlog.Warn(ctx, "uninstall request discarded. Helm revisions don't match")
```

Comment:
could we include the requested and current revision in this log line? we should directly include the result of `q.Get("revision")` just in case it didn't parse successfully (which seems unlikely given where it's coming from, but still)